README.md

```diff
@@ -2,6 +2,10 @@
 
 A daemon that manages a pool of block devices to create flexible filesystems.
 
+## Status
+
+March 8 2018: Stratis 0.5 released. See [release notes](docs/RELNOTES-0.5.md) for details.
```

Comment:
Hello from future! It's still 7th in most of the world ;)

Reply:
release will be tomorrow :)